extensions/wikia/Recirculation/services/ParselyDataService.class.php

```diff
@@ -52,16 +53,16 @@ function() use ( $type, $metadata, $count ) {
 	 */
 	private function apiRequest( $type, $meta, $count ) {
 		$options = [];
-		$options['limit'] = $count + self::PARSELY_API_EXTRA;
+		$options['limit'] = $count + static::PARSELY_API_EXTRA;
 
 		switch ( $type ) {
 			case 'vertical':
-				$options['sort'] = self::PARSELY_API_SORT;
+				$options['sort'] = static::PARSELY_API_SORT;
 				$options['pub_days'] = 30;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pub_days" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)